server.ts

```diff
@@ -0,0 +1,35 @@
+///<reference path='.\Declarations\node\node.d.ts' />
+
+import http = require("http");
+import AppInsights = require("./ApplicationInsights");
+
+AppInsights.setup("b7040ad8-a016-4057-903f-35edb58a6007")
+    .setAutoCollectRequestsEnabled(true)
+    .setAutoCollectPerformanceEnabled(true)
+    .enableVerboseLogging()
+    .start();
+
+AppInsights.instance.trackEvent("test event");
+AppInsights.instance.trackException(new Error());
+AppInsights.instance.trackMetric("test metric", 3);
+AppInsights.instance.trackTrace("test trace");
+
+var count = 0;
+var last = 0;
+var server = http.createServer((req: http.ServerRequest, res: http.ServerResponse) => {
+    count++;
+    var now = +new Date;
+    if(now - last > 1000) {
+        last = now;
+        console.log(count + " requests processed");
```

Comment:
Instead of using `console.log`, might be worth using a npm module such as `debug` (https://www.npmjs.com/package/debug); Express and some other middleware engines use it and make it easier to show/hide on demand with environment configuration.